docs/Guides/Delay-Accepting-Requests.md

```diff
@@ -0,0 +1,529 @@
+<h1 align="center">Fastify</h1>
+
+# Delay Accepting Requests
```

Comment:
Could you rename this to "A guide on how to use decorators and hooks"?

Reply:
@mcollina this is meant to solve #2500 by detailing what to consider in such a specific scenario. This is a much more advanced case than simply "how do I use decorators and hooks?". We should definitely have such a guide, but this one isn't it.

Reply:
I agree with @jsumners. I believe this is too specific to be considered a general-purpose guide on those topics. Specially because it lacks many of the features available by hooks and plugins.